tests/snuba/api/endpoints/test_organization_events_trends.py

```diff
@@ -537,6 +563,8 @@ def test_misery_trend_function(self):
             "minus_user_misery_range_2_user_misery_range_1": 1.0,
             "percentage_user_misery_range_2_user_misery_range_1": 2.0,
         }
+        for key, value in expected_data.items():
+            assert events["data"][0][key] == value
```

Comment:
You might want to consider a helper for this like `assert_event(events["data"][0], expected_data)`